### PR TITLE
Browse data

### PR DESCRIPTION
**Pull request by allenzhong**
- Browse data from root route
- Browse data with infinite scroll
- Testing with cucumber and rspec

### CODE REVIEW

File: vendor/assets/javascripts/jquery.infinitescroll.js [hunk 1]
```diff
@@ -0,0 +1,827 @@
+/*global jQuery: true */
```
Comment (chandleryang76) [inline vendor/assets/javascripts/jquery.infinitescroll.js:1]: Adding js into assets is okay, but here is no gem for jquery.infinitescroll?

Comment (allenzhong) [reply vendor/assets/javascripts/jquery.infinitescroll.js:1]: Actually, gem is just package which pack some js files in it.

Comment (allenzhong) [reply vendor/assets/javascripts/jquery.infinitescroll.js:1]: Normally, if we want to use 3rd party js or css lib without any modification or customization. Putting it in vendor folder is good choice.

**Review by JohnnyJ0622 — APPROVED** [review]
It's okay for merging.

**Pull request merged**